src/js/actions/tools.js

```diff
@@ -790,6 +808,37 @@ define(function (require, exports) {
             modifiers: {},
             fn: _vectorMaskHandler
         });
+        var DEBOUNCE_DELAY = 200;
+
+        // recreate pointer policies if we're in Vector Mask Mode on UI Store change events
+        _vectorMaskPolicyHandler = synchronization.debounce(function () {
+            var vectorMaskMode = toolStore.getVectorMode();
+
+            if (vectorMaskMode) {
+                var pointerPolicyID = toolStore.getVectorMaskPolicyID();
```

Comment:
I believe this handler should call a flux action instead of transferring.  I would suggest one `refreshVectorPolicy` action that would handle the transfers to the the remove/add policy actions.  Part of this is just that "it feels gross" to transfer from `beforeStartup` on a listener.  Paging Dr @iwehrman for a second opinion.